content/20.pharmaceuticals.md

```diff
@@ -522,16 +532,18 @@ Thus, Sigma-1 receptor activity may contribute to fluvoxamine's potential effect
 
 #### Potential and Limitations of High-Throughput Analyses
 
-These examples illustrate that computational research in drug repurposing can provide some valuable insights, but there are significant limitations to these methods.
-The urgency of the COVID-19 pandemic led many research groups to pivot towards this area of research without familiarity with best practices [@doi:10.1039/D0CS01065K].
+These examples illustrate that HTS-based evaluations of drug repurposing potential can provide some valuable insights, but there are significant limitations to these methods.
+The urgency of the COVID-19 pandemic led many research groups to pivot towards computational HTS research without familiarity with best practices in this area [@doi:10.1039/D0CS01065K].
 As a result, there is an excessive amount of information available from computational studies [@url:https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers], but not all of it is high-quality.
-Furthermore, it's not clear to what extent large-scale computational screening studies, especially those that are hypothesis free, produce insights that wouldn't be gained from smaller experimental screening efforts [@doi:10.1039/D0CS01065K; @doi:10.1126/science.abj9488].
+Additionally, much of the research used to identify and validate targets is not reproducible [@doi:10.1038/nrd3439-c1], presenting a possible limitation to target-based experimental screening and both hypothesis-driven and hypothesis-free computational HTS efforts. 
+Some efforts to repurpose antivirals have focused on host, rather than viral, proteins [@doi:10.1016/j.coi.2013.08.004], which might be expected to translate poorly _in vivo_ if the targeted proteins serve essential functions in the host.
+Furthermore, it's not clear to what extent large-scale computational HTS studies, especially those that are hypothesis free, produce insights that wouldn't be gained from smaller experimental screening efforts [@doi:10.1039/D0CS01065K; @doi:10.1126/science.abj9488].
```

Comment:
Does this primarily apply to computational _repurposing_ screens?  We don't talk about it, but computational methods that assess a billion synthesizable compounds are looking at a much different part of the chemical space than small experimental screens.

Reply:
I think I'm a little out of my depth here. I might remove this sentence because I don't think it provides anything super critical to making the point!
```suggestion
```